### PR TITLE
Fix PostgreSQL Metadata Source Tests

### PR DESCRIPTION
**Pull request by Chanakya-TS**
Updates Metadata Source Tests after changes from [commit](https://github.com/google/ml-metadata/commit/b0db34cefa074eb1d8fdb1aa9695f027af6e2f59)

Updates database creation and deletion commands (PSQL specific)  
## Description

Changes in [commit](https://github.com/google/ml-metadata/commit/b0db34cefa074eb1d8fdb1aa9695f027af6e2f59) no longer require EscapeString to be surrounded by single quotes.

Previously, databases cannot be created or deleted with the name "model-registry" due to "-". PR surrounds database name in double quotes.

## How Has This Been Tested?

Deployed model-registry-operator with a postgres server and ran `standalone_postgresql_metadata_source_test`. All 7 tests pass.

## Merge criteria:

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Comment by dhirajsb** [comment]
@isinyaaa can you also take a look?
/lgtm

File: ml_metadata/metadata_store/postgresql_metadata_source.cc [hunk 1]
```diff
@@ -252,7 +252,7 @@ absl::Status PostgreSQLMetadataSource::ConnectImpl() {
     MLMD_RETURN_IF_ERROR(databaseExistenceStatus);
     if (record_set.records_size() == 0) {
       const std::string create_database_cmd =
-          absl::Substitute("CREATE DATABASE $0;", config_.dbname().data());
+          absl::Substitute("CREATE DATABASE \"$0\";", config_.dbname().data());
```
Comment (antisaling) [inline ml_metadata/metadata_store/postgresql_metadata_source.cc:255]: IIUC, and also from the PR description, this fixes DB creation for names containing a "-", as you also raised on a previous meeting.
Can you split that into a separate commit, with the comment from the PR description? I believe it's important to keep track of the reason why certain changes were made.

IIRC we also need that fix on the MySQL metadata source `ConnectImpl`, can you confirm/test?

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+    "files.associations": {
+        "string": "cpp"
+    }
+}
```
Comment (antisaling) [inline .vscode/settings.json]: Nit: you could squash this change to keep the PR cleaner.

**Review by antisaling — CHANGES_REQUESTED** [review]
Thanks for the fix, Chanakya!
I have a [final nitpick](https://cbea.ms/git-commit) that I'd prefer if you split the two fixes on this PR in separate commits, because they kinda overlap although I think each requires a little more context, e.g. why are you removing quotes on one place and adding them on another? My point being that the changes are similar but for different reasons.
Because we're squash merging I guess you can also "copy-paste" the PR description on a single commit. That's just a click-saver to avoid getting into GitHub to try and figure out why the change was made (although the Mercurial syncs from upstream don't help much on this project).

TLDR: see comments inline, also https://cbea.ms/git-commit is a personal favorite -- especially useful in the open source world.
